pkgs/stdenv/generic/make-derivation.nix

```diff
@@ -77,8 +77,13 @@ rec {
 
     , ... } @ attrs:
 
-    # TODO(@Ericson2314): Make this more modular, and not O(n^2).
```

Comment:
I believe this about the hardening flags, but it's quite unclear so I just removed it.